docs/features/10-helm.md

```diff
@@ -16,7 +16,10 @@ provide a map of strings to set variables.
 // Login/Connect to your kubernetes cluster, where you want to deploy to.
 
 // First create a new helm instance and set the global parameters.
```

Comment:
Let's just improve the comment here:

// First create a new helm instance, optionally configure the deployment settings

Reply:
the "global parameters" in the old comment is quite misleading